packages/main/test/pages/base/AriaLabelHelper.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>AriaLabelHelper</title>
+    <script src="../../../bundle.esm.js" type="module"></script>
+</head>
+
+<style>
+    html,
+    body {
+        margin: 0;
+        padding: 0;
+        height: 100%;
+        width: 100%;
+    }
+
+    body {
+        padding: 1rem;
+        box-sizing: border-box;
+    }
+
+    .wrapper {
+        padding: 0.5rem;
+        box-sizing: border-box;
+        border: 1px #e1e1e1 dashed;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-direction: column;
+        margin: 0;
+    }
+
+    .field {
+        margin-top: 0.25rem;
+        display: block;
+        max-width: 100%;
+    }
+
+    .fields {
+        padding: 1rem 2rem;
+        margin-bottom: 2rem;
+        background-color: #e1e1e1;
+        border-bottom: 1px solid #888888;
+    }
+
+    .field,
+    .fields {
+        width: 85%;
+        max-width: 100%;
+    }
+
+    h4 {
+        text-align: start;
+        border-bottom: 1px solid #2f2f2f;
+    }
+
+
+</style>
+
+<body>
+    <div class="wrapper">
+        <div class="fields">
```

Comment:
This should also work for elements that are not ui5-label element yet I do not see this use case added in the test page, nor test for this. Also it is not correct to put inputs inside paragraphs and the whole point of the paragraph is not to align elements in a html page. Please use divisions or some other element which is used to group elements in a section.

Reply:
done

Reply:
This is not done. Paragraph elements are still present.